test/expected/bgw_db_scheduler.out

```diff
@@ -573,6 +573,83 @@ SELECT * FROM sorted_bgw_log;
       1 |    800000 | test_job_3_long  | After sleep job 3
 (11 rows)
 
+--Test sending a SIGHUP to a job
+\c :TEST_DBNAME :ROLE_SUPERUSER
+TRUNCATE bgw_log;
+SELECT ts_bgw_params_reset_time();
+ ts_bgw_params_reset_time 
+--------------------------
+ 
+(1 row)
+
+TRUNCATE _timescaledb_internal.bgw_job_stat;
+DELETE FROM _timescaledb_config.bgw_job;
+\c :TEST_DBNAME :ROLE_DEFAULT_PERM_USER
+--escalated priv needed for access to pg_stat_activity
+\c :TEST_DBNAME :ROLE_SUPERUSER
+SELECT ts_bgw_db_scheduler_test_run(-1);
+ ts_bgw_db_scheduler_test_run 
+------------------------------
+ 
+(1 row)
+
+SHOW timescaledb.shutdown_bgw_scheduler;
+ timescaledb.shutdown_bgw_scheduler 
+------------------------------------
+ off
+(1 row)
+
+ALTER SYSTEM SET timescaledb.shutdown_bgw_scheduler TO 'on';
+SELECT pg_reload_conf();
+ pg_reload_conf 
+----------------
+ t
+(1 row)
+
+SELECT pg_sleep(0.001);
+ pg_sleep 
+----------
+ 
+(1 row)
+
+SHOW timescaledb.shutdown_bgw_scheduler;
+ timescaledb.shutdown_bgw_scheduler 
+------------------------------------
+ on
+(1 row)
+
+SELECT ts_bgw_db_scheduler_test_wait_for_scheduler_finish();
+ ts_bgw_db_scheduler_test_wait_for_scheduler_finish 
+----------------------------------------------------
+ 
+(1 row)
+
+SELECT * FROM sorted_bgw_log;
+ msg_no |      mock_time      | application_name |                          msg                           
+--------+---------------------+------------------+--------------------------------------------------------
+      0 |                   0 | DB Scheduler     | [TESTING] Wait until 9223372036854775807, started at 0
+      1 | 9223372036854775807 | DB Scheduler     | bgw scheduler stopped due to shutdown_bgw guc
+(2 rows)
+
+ALTER SYSTEM SET timescaledb.shutdown_bgw_scheduler TO 'off';
+SELECT pg_reload_conf();
+ pg_reload_conf 
+----------------
+ t
+(1 row)
+
+SELECT pg_sleep(1);
```

Comment:
why do you need this long sleep?

Reply:
That's a typo, it should be the same as the other one.